docs/management/managing-licenses.asciidoc

```diff
@@ -0,0 +1,21 @@
+[[managing-licenses]]
+== License Management
+
+When you install Kibana, it generates a Basic license
+with no expiration date.  You can see the status of your
+license by opening *Management* and then clicking *License
+Management*.
+
+[role="screenshot"]
+image::images/management-license.png[]
+
+If you want to try the full set of
+link:https://www.elastic.co/subscriptions[Platinum features],
+you can start a 30-day trial.  You'll get access to security,
+machine learning, alerting, graphing capabilities, and more.
+
```

Comment:
If you change the above, I'd move this up before the image and say something like:

The 30-day trial enables you to try out the full set of https://www.elastic.co/subscriptions[Platinum features], including security, machine learning, alerting, graph capabilities, and more.

I think it should be "graph capabilities" or just "graph" -- "graphing" makes it sound like drawing graphs, whereas the graph app is about being able to treat your elasticsearch index like a graph database. It's confusing terminology in the context of Kibana, which is all about displaying data in graphs. Which "graph" lets you do, but in a specialized way that shows  the connections between the documents in your index. 

It works either way, but the "link:" prefix is only required if there are special characters or spaces in the URL.